src/sbs/instruments/acs.py

```diff
@@ -0,0 +1,194 @@
+from datetime import datetime
+import numpy as np
+import re
+import xarray as xr
+
+from src.sbs.core import NUM_PAT
```

Comment:
This may need to be changed to just sbs.core for review. I'm not sure I see why the sbs directory can't be the root for the package. I could probably change something in my IDE to fix import paths.

Reply:
This is sort of moot if NUM_PAT gets moved into acs.py, but it might be due to how the project is installed in your virtual env. If you install is as an editable install with the `-e` flag `py -m pip install -e .[dev]`, you should be able to use relative imports such as `from .core import ...`